lib/addons/aws-loadbalancer-controller/index.ts

```diff
@@ -69,7 +69,7 @@ const defaultProps: AwsLoadBalancerControllerProps = {
 
 function lookupImage(registry?: string, region?: string): Values {
     if (registry == null) {
-        console.log("Unable to get ECR repository for AWS Loadbalancer Controller for region " + region) + ". Using default helm image";
+        console.log("Unable to get ECR repository for AWS Loadbalancer Controller for region " + region + ". Using default helm image");
```

Comment:
minor: please add a dot at the end of the log message.

Reply:
@shapirov103 I have added dot at the end of the message.